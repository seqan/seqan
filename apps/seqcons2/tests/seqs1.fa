>simulated.1
GTTGACCCATGCAGAACGAGAGCTTGCCGCTGAATGGGAACATCGCCGACGTTAGAGATGGCCCCAGACAGGCATATCGTAAGTGATTTCGTAGCGAACC
>simulated.2
CGGTATGATCCAGTAGGTCCACGGAATCAAGGCGGCTCATGCGTGGATTGCCTTGACTTCCGTAGCTTGGGGGGTAGCCGGCACCGGCCCGGCTAACAGC
>simulated.3
TCAAGGCAATCCACGCATGAGCCGCCTTGATTCCGTGGACCAACTGGATCATACCGACCCTTAGAGCTGGGTAGGTTCGCTACGAAATCACTTACGATAT
>simulated.4
TACCCAGCTCTAAGGGTCGGTATGATCCAGTAGGTCCACGGAATCAAGGCGGCTCAGGCGTGGATTGCCTTGACTTCCGTAGCTTGGGGGGTAGCCGGCA
>simulated.5
AAGTGTATGGCACTCCCGCTGTTAGCCGGGCCGGTGCCGGCTACCCCCCAAGCTACGGAAGTCAAGGCAATCCACGCATGAGCCGCCTTGATTCCGTGGA
>simulated.6
TGCCAGAAGTGTATGGCACTCCCGCTGTTAGCCGGGCCGTTGCCGGCTACCCCCCAAGCTGCGGAAGTCAAGGCAATCCACGCATGAGCCGCCTTGATTC
>simulated.7
GTTAGAGATGGCCCCAGACAGGCATATCGTAAGTGATTTCGTAGCGAACCTACCCAGCTCTAAGGGTCGGTATGATCCAGTAGGTCCACGGAATCAAGGC
>simulated.8
CCCGCTGTTAGCCGGGCCGGTGCCGGCTACCCCCCAAGCTACGGAAGTCAAGGCAATCCACGAATGAGCCGCCTTGATTCCGTGGACCTACTGGATCATA
>simulated.9
CGGCGATGTTCCCATTCAGCGGCAAGCTCTCGTTCTGCATGGGTCAACTCCCTCACGAAGAAATATAGTCTGTAAGCGCAAGGGATAGGGCCGACACACC
>simulated.10
CCGGGCCGGTGCCGGCTACCCCCCAAGCTACGGAAGTCAAGGCAATCCACGCATGAGCCGCCTTGATTCCGTGGACCTACTGGATCATACCGACCCTTAG
>simulated.11
TTGCCGCTGAATGGGAACATCGCCGACGTTAGAGATGGCCCCAGACAGGCATATCGTAAGTGATTTCGTAGCGAACCTACCCAGCTCTAAGGGTCGGTAT
>simulated.12
GCCGGGCCGGTGCCGGCTACCCCCCAAGCTACGGAAGTCAAGGCAATCCACGCATGAGCCGCCTTGATTCCGTGGACCTACTGGATCATACCGACCCTTA
>simulated.13
CGTAAGTGATTTCGTAGCGAACCTACCCAGCTCTAAGGGTCGGTCTGATCCAGTAGGTCCACGGAATCAAGGCGGCTCATGCGTGGATTGCCTTGACTTC
>simulated.14
TTGCGCTTACAGACTTTATTTCTTCGTGAGGGAGTTGACCCATGCAGAACCAGAGCTTGCCGCTGAATGGGAACATCGCCGACGTTAGAGATGGCACCAG
>simulated.15
GCACTCCCGCTGTTAGCCGGGCCGGTGCCGGCTACCCCCCAAGCTACGGAAGTCAAGGCAATCCACGCATGAGCGGCCTTGATTCCGTGGACCTACTGGA
>simulated.16
GAGTTGACCCATGCAGAACGAGAGCTTGCCGCTGAATGGGAACATCGCCGACGTTAGAGATGGCCCCAGACAGGCATATCGTAAGTGATTTCGTAGCGAA
>simulated.17
CCATGCAGAACGAGAGCTTGCCGCTGAATGGGAACATCGCCGACGTTAGAGATGGCCCCAGACAGGCATATCGTAAGTGATTTCGTAGCGAACCTACCCA
>simulated.18
CCTACTGGATCATACCGACCCTTAGAGCTGGGTAGGTTCGCTACGAAATCACTTACGATATGCCTGTCTGGGGCCATCTCTAACGTCGGCGATTTTCCCA
>simulated.19
CTCTAACGTCGGCGATGTTCCCATTCAGCGGCAAGCTCTCGTTCTGCATGGGTCAACTCCCTCACGAAGAAATAAAGTCTGTAAGCGCAAGGGATAGGGC
>simulated.20
TTAGCCGGGCCGGTGCCGGCTACCCCCCAAGCTACGGAAGTCAAGGCAATCCACGCATGAGCCGCCTTGATTCCGTGGACCTACTGGATCATACCGACCC
