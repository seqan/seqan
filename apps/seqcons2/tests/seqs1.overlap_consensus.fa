>consensus_0
TTGCGCTTACAGACTTTATTTCTTCGTGAGGGAGTTGACCCATGCAGAACGAGAGCTTGCCGCTGAATGG
GAACATCGCCGACGTTAGAGATGGCCCCAGACAGGCATATCGTAAGTGATTTCGTAGCGAACCTACCCAG
CTCTAAGGGTCGGTATGATCCAGTAGGTCCACGGAATCAAGGCGGCTCATGCGTGGATTGCCTTGACTTC
CGTAGCTTGGGGGGTAGCCGGCACCGGCCCGGCTAACAGC
>consensus_1
TGCCAGAAGTGTATGGCACTCCCGCTGTTAGCCGGGCCGGTGCCGGCTACCCCCCAAGCTACGGAAGTCA
AGGCAATCCACGCATGAGCCGCCTTGATTCCGTGGACCTACTGGATCATACCGACCCTTAGAGCTGGGTA
GGTTCGCTACGAAATCACTTACGATATGCCTGTCTGGGGCCATCTCTAACGTCGGCGATTTTCCCA
>consensus_2
CTCTAACGTCGGCGATGTTCCCATTCAGCGGCAAGCTCTCGTTCTGCATGGGTCAACTCCCTCACGAAGA
AATAAAGTCTGTAAGCGCAAGGGATAGGGCCGACACACC
