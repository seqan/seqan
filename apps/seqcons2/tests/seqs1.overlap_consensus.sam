@HD	VN:1.4	SO:unsorted
@PG	ID:SeqAn
@SQ	SN:consensus_1	LN:250
@SQ	SN:contig_0	LN:206
@SQ	SN:contig_1	LN:109
simulated.14	0	consensus_1	1	255	100M	*	0	0	TTGCGCTTACAGACTTTATTTCTTCGTGAGGGAGTTGACCCATGCAGAACCAGAGCTTGCCGCTGAATGGGAACATCGCCGACGTTAGAGATGGCACCAG	]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]	NM:i:2	MD:Z:50G44C4
simulated.16	0	consensus_1	32	255	100M	*	0	0	GAGTTGACCCATGCAGAACGAGAGCTTGCCGCTGAATGGGAACATCGCCGACGTTAGAGATGGCCCCAGACAGGCATATCGTAAGTGATTTCGTAGCGAA	]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]	NM:i:0	MD:Z:100
simulated.1	0	consensus_1	34	255	100M	*	0	0	GTTGACCCATGCAGAACGAGAGCTTGCCGCTGAATGGGAACATCGCCGACGTTAGAGATGGCCCCAGACAGGCATATCGTAAGTGATTTCGTAGCGAACC	]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]	NM:i:0	MD:Z:100
simulated.17	0	consensus_1	40	255	100M	*	0	0	CCATGCAGAACGAGAGCTTGCCGCTGAATGGGAACATCGCCGACGTTAGAGATGGCCCCAGACAGGCATATCGTAAGTGATTTCGTAGCGAACCTACCCA	]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]	NM:i:0	MD:Z:100
simulated.11	0	consensus_1	57	255	100M	*	0	0	TTGCCGCTGAATGGGAACATCGCCGACGTTAGAGATGGCCCCAGACAGGCATATCGTAAGTGATTTCGTAGCGAACCTACCCAGCTCTAAGGGTCGGTAT	]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]	NM:i:0	MD:Z:100
simulated.7	0	consensus_1	84	255	100M	*	0	0	GTTAGAGATGGCCCCAGACAGGCATATCGTAAGTGATTTCGTAGCGAACCTACCCAGCTCTAAGGGTCGGTATGATCCAGTAGGTCCACGGAATCAAGGC	]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]	NM:i:0	MD:Z:100
simulated.13	0	consensus_1	111	255	100M	*	0	0	CGTAAGTGATTTCGTAGCGAACCTACCCAGCTCTAAGGGTCGGTCTGATCCAGTAGGTCCACGGAATCAAGGCGGCTCATGCGTGGATTGCCTTGACTTC	]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]	NM:i:1	MD:Z:44A55
simulated.4	0	consensus_1	134	255	100M	*	0	0	TACCCAGCTCTAAGGGTCGGTATGATCCAGTAGGTCCACGGAATCAAGGCGGCTCAGGCGTGGATTGCCTTGACTTCCGTAGCTTGGGGGGTAGCCGGCA	]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]	NM:i:1	MD:Z:56T43
simulated.2	0	consensus_1	151	255	100M	*	0	0	CGGTATGATCCAGTAGGTCCACGGAATCAAGGCGGCTCATGCGTGGATTGCCTTGACTTCCGTAGCTTGGGGGGTAGCCGGCACCGGCCCGGCTAACAGC	]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]	NM:i:0	MD:Z:100
simulated.6	0	contig_0	1	255	100M	*	0	0	TGCCAGAAGTGTATGGCACTCCCGCTGTTAGCCGGGCCGTTGCCGGCTACCCCCCAAGCTGCGGAAGTCAAGGCAATCCACGCATGAGCCGCCTTGATTC	]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]	NM:i:2	MD:Z:39G20A39
simulated.5	0	contig_0	7	255	100M	*	0	0	AAGTGTATGGCACTCCCGCTGTTAGCCGGGCCGGTGCCGGCTACCCCCCAAGCTACGGAAGTCAAGGCAATCCACGCATGAGCCGCCTTGATTCCGTGGA	]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]	NM:i:0	MD:Z:100
simulated.15	0	contig_0	16	255	100M	*	0	0	GCACTCCCGCTGTTAGCCGGGCCGGTGCCGGCTACCCCCCAAGCTACGGAAGTCAAGGCAATCCACGCATGAGCGGCCTTGATTCCGTGGACCTACTGGA	]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]	NM:i:1	MD:Z:74C25
simulated.8	0	contig_0	21	255	100M	*	0	0	CCCGCTGTTAGCCGGGCCGGTGCCGGCTACCCCCCAAGCTACGGAAGTCAAGGCAATCCACGAATGAGCCGCCTTGATTCCGTGGACCTACTGGATCATA	]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]	NM:i:1	MD:Z:62C37
simulated.20	0	contig_0	28	255	100M	*	0	0	TTAGCCGGGCCGGTGCCGGCTACCCCCCAAGCTACGGAAGTCAAGGCAATCCACGCATGAGCCGCCTTGATTCCGTGGACCTACTGGATCATACCGACCC	]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]	NM:i:0	MD:Z:100
simulated.12	0	contig_0	31	255	100M	*	0	0	GCCGGGCCGGTGCCGGCTACCCCCCAAGCTACGGAAGTCAAGGCAATCCACGCATGAGCCGCCTTGATTCCGTGGACCTACTGGATCATACCGACCCTTA	]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]	NM:i:0	MD:Z:100
simulated.10	0	contig_0	32	255	100M	*	0	0	CCGGGCCGGTGCCGGCTACCCCCCAAGCTACGGAAGTCAAGGCAATCCACGCATGAGCCGCCTTGATTCCGTGGACCTACTGGATCATACCGACCCTTAG	]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]	NM:i:0	MD:Z:100
simulated.3	0	contig_0	68	255	100M	*	0	0	TCAAGGCAATCCACGCATGAGCCGCCTTGATTCCGTGGACCAACTGGATCATACCGACCCTTAGAGCTGGGTAGGTTCGCTACGAAATCACTTACGATAT	]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]	NM:i:1	MD:Z:41T58
simulated.18	0	contig_0	107	255	100M	*	0	0	CCTACTGGATCATACCGACCCTTAGAGCTGGGTAGGTTCGCTACGAAATCACTTACGATATGCCTGTCTGGGGCCATCTCTAACGTCGGCGATTTTCCCA	]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]	NM:i:0	MD:Z:100
simulated.19	0	contig_1	1	255	100M	*	0	0	CTCTAACGTCGGCGATGTTCCCATTCAGCGGCAAGCTCTCGTTCTGCATGGGTCAACTCCCTCACGAAGAAATAAAGTCTGTAAGCGCAAGGGATAGGGC	]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]	NM:i:0	MD:Z:100
simulated.9	0	contig_1	10	255	100M	*	0	0	CGGCGATGTTCCCATTCAGCGGCAAGCTCTCGTTCTGCATGGGTCAACTCCCTCACGAAGAAATATAGTCTGTAAGCGCAAGGGATAGGGCCGACACACC	]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]]	NM:i:1	MD:Z:65A34
