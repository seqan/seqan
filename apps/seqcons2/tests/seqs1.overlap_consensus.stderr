SeqCons
=======

__OPTIONS____________________________________________________________________

VERBOSITY              	NORMAL
OPERATION              	overlap MSA + consensus

INPUT FILE             	seqs1.fa
OUTPUT READS           	seqs1.overlap_consensus.sam
OUTPUT CONSENSUS       	seqs1.overlap_consensus.fa

OVERLAP MIN LENGTH     	20
OVERLAP MAX ERR-RATE   	5
OVERLAP MIN COUNT      	3
OVERLAP WINDOW SIZE    	20

K-MER SIZE             	20
K-MER MAX OCCURENCES   	200

REALIGNMENT BANDWIDTH  	10
REALIGNMENT ENVIRONMENT	20

__LOADING DATA_______________________________________________________________

Loading reads from seqs1.fa...OK

__COMPUTATION________________________________________________________________

Performing consensus computation... OK
Performing realignment... OK

__WRITING RESULT_____________________________________________________________

Writing consensus to seqs1.overlap_consensus.fa ... OK
Writing alignments to seqs1.overlap_consensus.sam ... OK

Overall time: 0.781774 s

Done.  Have a nice day.
